src/Activities/CustomActivityDimensions/ICustomTagObjectsDimensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Omex.Extensions.Activities
+{
+	/// <summary>
+	/// Define custom metric dimensions which are extracted from Activity TagObjects  by <see cref="ActivityMetricsSender.SendActivityMetric"/>
+	/// Make sure that the value of any custom dimensions are not random values such as ID, timestamp because it will lead to metric explosion.
+	/// </summary>
+	public interface ICustomTagObjectsDimensions
```

Comment:
Would it make sense to rename CustomTagObjectsDimensions to CustomTagDimensions?
It seems more readable to me.
If not don't worry about it.

Reply:
We use Activity.TagsObjects , so I want to keep them uniform in variable name and object field name